tests/test_dle.py

```diff
@@ -610,6 +610,76 @@ def test_incs_no_atomic(self):
         assert 'collapse(1)' in str(op1)
         assert 'atomic' not in str(op1)
 
+    @pytest.mark.parametrize('exprs,simd_level', [
```

Comment:
out of curiosity, do we really need all these cases? or are some of them actually nearly redundant?

Reply:
hm... I would not say that all of them are "difficult corner cases", however, they are all different.
Just added resulting data asserts as well.

Reply:
Dropped another that should be dropped.